src/cs/usfca/edu/histfavcheckout/filters/AnalyticsFilter.java

```diff
@@ -36,16 +35,15 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 			@Override
 			public void run() {
 				EDRRequest edrRequest = new EDRRequest(req.getMethod(), req.getPathInfo(),
-						System.currentTimeMillis() - timestamp, String.valueOf(res.getStatus()), SERVICE_NAME, 
+						System.currentTimeMillis() - timestamp, String.valueOf(res.getStatus()), SERVICE_NAME,
 						success, String.valueOf(timestamp), "");
 				try {
 					boolean success = APIClient.sendEDR(edrRequest);
 					if(success) {
-						//System.out.println("EDR Event responded with non 200 responseCode: " + responseCode + " response!");
-						// LOG here EDR responded non 200 status and log what status was returned. 
+						System.out.println("EDR Event responded with non 200 responseCode: " + res.getStatus() + " response!");
```

Comment:
As specified earlier. Here, EDR is returning success so we should not log that EDR is returning non 200 response code. 

Please replace the selected lines with below, 

```
if(!success) {
    System.out.println("EDR Event responded with responseCode: " + res.getStatus());
}
```